packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -637,8 +663,27 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
       e: React.SyntheticEvent,
       searchInputProps: DropdownSearchInputProps,
     ) => {
-      if (keyboardKey.getCode(e) === keyboardKey.Tab && !_.isNil(highlightedIndex)) {
-        selectItemAtIndex(highlightedIndex)
+      switch (keyboardKey.getCode(e)) {
```

Comment:
Please please create an issue for creating dropdownBehavior. It's really hard to keep track of everything that it happening here.

Reply:
@silviuavram  we can pair on creating behavior, to show how the behaviors work and action handlers are assigned.

Reply:
https://github.com/stardust-ui/react/issues/875